.travis.yml

```diff
@@ -0,0 +1,45 @@
+dist: trusty
+
+cache:
+  apt: true
+  directories:
+    # Cache directory for older Composer versions.
+    - $HOME/.composer/cache/files
+    # Cache directory for more recent Composer versions.
+    - $HOME/.cache/composer/files
+
+language:
+  - php
+
+php:
+  - 5.6
+  - 7.0
+  - 7.1
+  - 7.2
+  - 7.3
+  - "7.4snapshot"
+
+matrix:
+  fast_finish: true
+  include:
+    - php: 7.3
+    - php: 7.2
+    - php: 7.1
+    - php: 7.0
+    - php: 5.6
+
+  allow_failures:
+    # Allow failures for unstable builds.
+    - php: "7.4snapshot"
+
+before_install:
+    - composer install --dev
+
+script:
+  # Lint the PHP files against parse errors.
+  - if find . -path ./vendor -prune -o -path ./bin -prune -o -name "*.php" -exec php -l {} \; | grep "^[Parse error|Fatal error]"; then exit 1; fi
+
+  # Validate the composer.json file.
+  # @link https://getcomposer.org/doc/03-cli.md#validate
+  - composer validate --no-check-all --strict
+  - composer lint .
```

Comment:
Make this script dependent on the `ENV` variable set in the matrix, like so:
`- if [[ "$PHPCS" == 1 ]]; then composer lint; fi`.

Move the `.` to the `phpcs.xml.dist` ruleset.

You don't need PHPCS to run against every single PHP version. All PHPCS sniffs are unit tested to give the same results across PHP versions, so you only need to run PHPCS on one PHP version - preferably a high one, such as PHP 7.3 as suggested above -.

No need to repeat the unit tests from PHPCS and the external standards themselves.